ably/proto_message_decoding_test.go

```diff
@@ -107,6 +107,34 @@ func loadMsgpackFixtures() ([]MsgpackTestFixture, error) {
 	return o, err
 }
 
+func Test_msgpackExtrasJsonCompatible(t *testing.T) {
```

Comment:
```suggestion
func TestMsgpackExtrasJsonCompatible(t *testing.T) {
```

(to be consistent with other test names)